main.cpp

```diff
@@ -52,84 +52,104 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun: sun
+//  action 1: the sun heats earth
+sun.heatEarth();
 
-//  action 2:
+//  action 2: the sun shines bright
+sun.shineBright();
+
+
+//  action 3: the sun sets
+sun.set();
 
-//  action 3:
- 
 //  2)
-//  Noun:
-//  action 1:
+//  Noun: apple
+//  action 1: apple ripens
+apple.ripen(); 
 
-//  action 2:
+//  action 2: apple falls
+apple.fall();
 
-//  action 3:
+//  action 3: apple rots
+apple.rot();
  
 //  3)
-//  Noun:
-//  action 1:
+//  Noun: traffic
+//  action 1: traffic disperses
+traffic.disperse();
 
-//  action 2:
+//  action 2: traffic flows steadily
+traffic.flowSteadily();
 
-//  action 3:
+//  action 3: traffic heats the road
+traffic.heatRoad();
  
 //  4)
-//  Noun:
-//  action 1:
+//  Noun: kid
+//  action 1:  kid wakes up parents
+kid.wakeParents();
 
-//  action 2:
+//  action 2: kid walks to school
+kid.walkToSchool();
 
-//  action 3:
+//  action 3: kid refuses to listen
+kid.refuseToListen();
  
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: lion
+//  action 1: a lion roars
+lion.roar();
+//  action 2: a lion hunts
+lion.hunt();
+//  action 3: a lion eats
+lion.eat();
  
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: earth
+//  action 1: earth rotates
+earth.rotate();
+//  action 2: contains atmosphere with gravity
+earth.containAtmosphere();
+//  action 3: earth waxes and wanes
+earth.waxAndWane();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.